addons/website/models/ir_ui_view.py

```diff
@@ -522,7 +522,19 @@ def save(self, value, xpath=None):
                 ('website_id', '=', current_website.id)
             ], limit=1)
             if website_specific_view:
-                self = website_specific_view
+                if (
+                    website_specific_view.page_ids
+                    and website_specific_view.page_ids[0].url != self.page_ids[0].url
```

Comment:
Not sure why we do but we do have `first_page_id` which maybe is made for accessing the only page linked to the view? :thinking:

Reply:
We need this field to be able to add the (clickable) relationship inside the form view of an `ir.ui.view` to it's `website.page`.
Due to the nature of our ORM, it's a o2m (inverse of the m2o), even if it does not make sense to have multiple page for a given view, the ORM allows it (and never see a point to try to write custom code to prevent it).

It's only used for this at the moment but indeed we can probably use it here